### PR TITLE
Use unchecked block in for loop to save gas

### PR DESCRIPTION
**Pull request by KurtMerbeth**

### CODE REVIEW

**Review by aminlatifi — COMMENTED** [review]
@KurtMerbeth Please run `forge fmt` before commit. Other than the rest looks good

File: contracts/GIVpower.sol [hunk 1]
```diff
@@ -109,32 +109,32 @@ contract GIVpower is GardenUnipoolTokenDistributor, IERC20MetadataUpgradeable {
...
-                // && _roundBalance._releasablePowerAmount == 0
-                continue;
-            }
+            if (_roundBalance.unlockableTokenAmount > 0) {
+                uint256 _releasablePowerAmount = _roundBalance.releasablePowerAmount;
+                uint256 _unlockableTokenAmount = _roundBalance.unlockableTokenAmount;
 
-            uint256 _releasablePowerAmount = _roundBalance.releasablePowerAmount;
-            uint256 _unlockableTokenAmount = _roundBalance.unlockableTokenAmount;
+                _userLock.totalAmountLocked = _userLock.totalAmountLocked.sub(_unlockableTokenAmount);
```
Comment (aminlatifi) [inline contracts/GIVpower.sol:122]: @KurtMerbeth Do you think it's worth and safe to make this one unchecked too?

Comment (KurtMerbeth) [reply contracts/GIVpower.sol:122]: i don't see a situation where `_totalAmountLocked < _unlockableTokenAmount` is. So it should be secure and safe some gas

Comment (aminlatifi) [reply contracts/GIVpower.sol:122]: Essentially `_userLock.totalAmountLocked` must be bigger than `_unlockableTokenAmount` if the rest of the code works correctly. But you are right, let's leave it for safety

**Review by aminlatifi — APPROVED** [review]

**Comment by aminlatifi** [comment]
Thanks @KurtMerbeth

**Pull request merged**